cypress/fixtures/marriage/expected-marriage-records.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+const search = {
+  systemNumber: '',
+  forenames: 'Tester',
+  surname: 'MULTIPLE',
+  day: '29',
```

Comment:
Would prefer the date to look like this

dom: {
day: '29',
month: '02',
year: '2012'
}

and change subsequent usage accordingly.

Reply:
Done